submissions/Levi123/oop-exercise/index.js

```diff
@@ -0,0 +1,90 @@
+// class Animal {
+//    constructor (species, name, gender){
+//       this.species = species;
+//       this.name = name;
+//       this.gender = gender;
+//    }
+// }
+
+class Entity {
+   constructor (name, species, gender, saying){
+      this.name = name;
+      this.species = species;
+      this.gender = gender;
+      this.saying = saying;
+   }
+
+   getProps (){
+      return [
+         this.name,
+         this.species,
+         this.gender,
+         this.saying,
+      ]
+   }
+}
+
+class Human extends Entity {
+   constructor (name, gender, saying){
+      super(name, 'Human', gender, saying);
+      this.legs = 2;
+      this.hands = 2;
+   }
+
+   getProps (){
+      return super.getProps().concat([this.legs, this.hands])
+   }
+}
+
+class Woman extends Human {
+   constructor(name, saying){
+      super(name, 'Woman', saying)
```

Comment:
Men and women are the same species.

Reply:
Yes, a man and a woman get one species - a person. But it has a gender property to display gender, if that's what you're after.